src/itemChoice/index.tsx

```diff
@@ -123,10 +123,12 @@ class ItemChoice extends PureComponent <ItemChoiceProps> {
           </div>
         )}
         {children}
-        {rightAddon && (
-          <div className={cc(prefix({ 'itemChoice-rightAddon': true }))}>{rightAddon}</div>
-        )}
-        {rightIcon}
+        <div className="kirk-itemChoice-right">
+          {rightAddon && (
```

Comment:
Is it intended to have an empty `div` in some cases ?

Reply:
It's never empty, we always have the arrow as default.

Reply:
rightAddon is when we want to add a content near the arrow.

Reply:
right, my bad, I didn't see `{rightIcon}` underneath